torch_np/tests/numpy_tests/lib/test_function_base.py

```diff
@@ -1123,7 +1122,7 @@ def test_basic(self):
         assert_array_almost_equal(z, zo, 11)
 
 
-@pytest.mark.xfail(reason='TODO: implement')
+@pytest.mark.skip(reason='ignore trim_zeros')
```

Comment:
Weird function, acting on an arbitary sequence, not not seem to really belong to core numpy, am suspecting not really used: https://github.com/numpy/numpy/blob/v1.24.0/numpy/lib/function_base.py#L1800-L1850
On my personal scale it's well on the "easier to implement yourself" side.
Can add --- basically copy-paste the numpy implementation---if you think it's useful.

Reply:
Then let's just leave it as `xfail` as it could potentially be implemented, only that we don't cover it yet. 
The NumPy implementation seems terribly slow, but it can have a parallel implementation calling a few kernels. (e.g. via a variation of https://discuss.pytorch.org/t/first-nonzero-index/24769/10 or a different trick).

That being said, I agree that this is very low prio, but I would still leave it as an xfail.

Reply:
OK, done in  https://github.com/Quansight-Labs/numpy_pytorch_interop/pull/48/commits/1417e8c743be2ed0084cea59b8688cafe237c2f9

I was going to trawl through xfails to convert more xfails to skips, but OK, let's not do it, not now anyway.